prime-factors/prime-factors_test.lua

```diff
@@ -3,47 +3,47 @@ local primeFactors = require('prime-factors')
 describe('primeFactors', function()
 
   it('returns an empty array for 1', function()
-    assert.are.same({}, PrimeFactors(1))
+    assert.are.same({}, primeFactors(1))
```

Comment:
Note that above the source file is `require`d and saved into `primeFactors`, not `PrimeFactors`